libcloud/compute/base.py

```diff
@@ -1260,8 +1260,9 @@ def delete_key_pair(self, key_pair):
         raise NotImplementedError(
             'delete_key_pair not implemented for this driver')
 
-    def wait_until_running(self, nodes, wait_period=3, timeout=600,
-                           ssh_interface='public_ips', force_ipv4=True):
+    def wait_until_running(self, nodes, wait_period=3,
+                           timeout=600, ssh_interface='public_ips',
+                           force_ipv4=True, **ex_list_nodes_kwargs):
```

Comment:
There should be no ** here since user should pass in a dict to the function. I will remove it when merging the patch.